src/chapters/booleans-and-conditionals/booleans.rst

```diff
@@ -0,0 +1,209 @@
+.. _booleans:
+
+Booleans
+========
+
```

Comment:
From the objectives: "Use ``Boolean`` to convert non-boolean values". 

Is there ever an explicit concept check for this idea?

Reply:
There sort of is, in the form of a Try It! block with a Question on the same page. I do wonder if we are using these two formats together effectively, and how we should be differentiating them. I put this on the agenda for our next meeting.